src/clj/rems/db/category.clj

```diff
@@ -0,0 +1,82 @@
+(ns rems.db.category
+  (:require [clojure.tools.logging :as log]
+            [rems.db.core :as db]
+            [rems.json :as json]
+            [rems.schema-base :as schema-base]
+            [rems.common.util :refer [build-index]]
+            [schema.coerce :as coerce]
+            [schema.core :as s]
+            [medley.core :refer [assoc-some]]))
+
+(s/defschema CategoryData
+  {(s/optional-key :description) schema-base/LocalizedString
```

Comment:
would it make more sense to have `description` as mandatory field?

also i think this is also missing `title` field, now that i'm thinking about what we discussed earlier

Reply:
I think optional is good, because some user may want to leave it blank and not have anything displayed.

Reply:
`:title` would be good, required, `LocalizedString`